cmd/krel/templates/latest/rpm/kubeadm/kubeadm.spec

```diff
@@ -24,37 +26,30 @@ BuildRequires: systemd-rpm-macros
 %endif
 
 %description
-Command-line utility for administering a Kubernetes cluster.
+%{summary}.
 
 %prep
-%setup -c -D -T -a 0 -n kubeadm
+%setup -q -c
 
 %build
+# Nothing to build
 
 %install
-
+# Detect host arch
 KUBE_ARCH="$(uname -m)"
 
-cd %{_builddir}/kubeadm/
+# Install files
 mkdir -p %{buildroot}%{_bindir}
-mkdir -p %{buildroot}%{_sysconfdir}/kubernetes/manifests/
 mkdir -p %{buildroot}%{_unitdir}/kubelet.service.d/
 
-install -p -m 755 -t %{buildroot}%{_bindir}/ ${KUBE_ARCH}/kubeadm
-install -m 755 -d %{buildroot}%{_sysconfdir}/kubernetes/manifests/
-install -p -m 644 -t %{buildroot}%{_unitdir}/kubelet.service.d/ 10-kubeadm.conf
+install -p -m 755 ${KUBE_ARCH}/kubeadm %{buildroot}%{_bindir}/kubeadm
+install -p -m 644 10-kubeadm.conf %{buildroot}%{_unitdir}/kubelet.service.d/10-kubeadm.conf
 
 %files
 %{_bindir}/kubeadm
 %dir %{_unitdir}/kubelet.service.d
```

Comment:
Does this directory need a dot file to keep it on debian?

Reply:
@puerco I'm not sure to what line is this comment referring but if we're talking about `%{buildroot}%{_unitdir}/kubelet.service.d/10-kubeadm.conf`, `10-kubeadm.conf` is a regular file, so no dot file is needed. Dot file is needed only to keep an empty directory.

Reply:
This is about ` %{_unitdir}/kubelet.service.d`

Reply:
Oops sorry I just saw the next line after you mentioned it  :)